src/media/js/views/search.js

```diff
@@ -116,7 +116,7 @@ define('views/search',
             ':terms': 'terms',
         };
         var $q = $('#search-q');
-        var query = $q.val();
+        var query = $q.val().trim().replace(/  +/g, ' ');
```

Comment:
this probably belongs somewhere in https://github.com/mozilla/marketplace-core-modules/blob/bf20927/core/utils.js